pages/index.js

```diff
@@ -1,4 +1,5 @@
-import Head from 'next/head'
+import Head from "next/head";
+import { DataFetcher, FakeDbStrategy } from "./api";
```

Comment:
This import seems unnecessary. Please remove this file from the PR. Let me show you how to do this in a video

Reply:
https://www.loom.com/share/4dcdaa9e7c084e5db4207118c5ff51a5

The command I want you to run is simply:
```
gco master -- pages/index.js
```

This is a very useful concept to know though!